worldwind-tutorials/src/commonMain/kotlin/earth.worldwind.tutorials/PathsTutorial.kt

```diff
@@ -33,6 +37,9 @@ class PathsTutorial(private val engine: WorldWind) : AbstractTutorial() {
                     Position.fromDegrees(40.0, -40.0, 0.0)
                 )
             ).apply {
+                attributes.apply {
```

Comment:
Same as above

Reply:
It will suffice, maybe this change should be removed there's already line in this tutorial with width greater than 1